### ISSUE TITLE
Gambling items not counted in total cash value when above 100 credit

### ISSUE THREAD

**Issue by Jake-Breaks-Stuff** [JunyDeveloper/GamblingModLethalCompany#50]
I've tested this a handful of times now, and my friends and I have replicated this bug a handful of times.

We've tested with gambling on items (scrap metal and keys if the item type matters and our closest to value 96), getting their values under 100 and submitting them for the quota and having them count.

Then on 3 separate occasions we've had items over 100 credits, and had them not count towards the quota. (110, 108, and 126 credit keys and tattered metal).

TLDR
Items with their values gambled up to over 100 credits are ignored and treated as worth 0 by the company.

**Comment by MGJerry** [comment]
Duplicate of #27
